scripts/utils.js

```diff
@@ -36,11 +42,28 @@ module.exports = {
    * it is seen in simple-icons.json)
    * @param {String} htmlFriendlyTitle The title to convert
    */
+
   htmlFriendlyToTitle: (htmlFriendlyTitle) =>
     htmlFriendlyTitle
       .replace(/&#([0-9]+);/g, (_, num) => String.fromCharCode(parseInt(num)))
       .replace(
         /&(quot|amp|lt|gt);/g,
         (_, ref) => ({ quot: '"', amp: '&', lt: '<', gt: '>' }[ref]),
       ),
+
+  /**
+   * Converts a brand title (as it is seen in simple-icons.json) into a brand
+   * title in HTML/SVG friendly format.
+   * @param {String} brandTitle The title to convert
+   */
+  titleToHtmlFriendly: (brandTitle) =>
+    brandTitle
+      .replace(/"/g, '&quot;')
+      .replace(/&/g, '&amp;')
```

Comment:
The current replacements will change `"` into `&amp;quot;` [^1], it should be the following instead.

```suggestion
      .replace(/&/g, '&amp;')
      .replace(/"/g, '&quot;')
```

[^1]: You can run `"\"".replace(/"/g, '&quot;').replace(/&/g, '&amp;')` in Node to verify this.